misc/dist/html/full-size.html

```diff
@@ -52,6 +52,20 @@
 	margin: auto;
 }
 
+#status-splash.no-show-img {
+	display: none;
+}
+
+#status-splash.fullsize {
+	height: 100%;
+	width: 100%;
+	object-fit: contain;
+}
+
+#status-splash.nearest-neighbor {
+	image-rendering: pixelated;
+}
+
```

Comment:
My suggestion on how to have the classes as the "positive" value.

```suggestion
	margin: auto;
	display: none;
	image-rendering: pixelated;
}

#status-splash.show-image {
	display: block;
}

#status-splash.fullsize {
	height: 100%;
	width: 100%;
	object-fit: contain;
}

#status-splash.use-filter {
	image-rendering: auto;
}

```